packages/bot/src/stores/toolbar-store.js

```diff
@@ -36,6 +36,7 @@ export default class ToolbarStore {
                 const scroll_distance = toolbox_width - block_canvas_rect.left + toolbox.width;
                 scrollWorkspace(workspace, scroll_distance, true, false);
             }
+            this.root_store.core.gtm.pushDataLayer({ event: 'DBot - Toolbox is visible', value: true });
```

Comment:
@anton-binary can we use snake_case for event names? it will be the same as other places, `bot_toolbox_visible` or st like this.

Reply:
done